README.md

```diff
@@ -0,0 +1,57 @@
+# Sugar 🇦🇮
+
+Sugar AI is a **central hub for all AI-powered chatbots and assistants** integrated within Sugar. Designed to enhance interactive learning, it **brings the power of AI and LLMs** into various educational activities, making learning more engaging and accessible for children.
+By seamlessly integrating AI into Sugar activities, Sugar AI fosters creativity, problem-solving, and interactive learning. Children can engage with AI chatbots for conversations, receive coding assistance through intelligent programming guides, and explore new concepts in an intuitive and friendly environment.
+
+## Current AI Models🤖
+
+### 1. Chatbot for the [Chat Activity](https://github.com/sugarlabs/chat)💬
+An interactive AI that helps kids learn through engaging conversations.
+
+### 2. AI Coding Assistant for the [Pippy Activity](https://github.com/sugarlabs/Pippy)👩‍💻
+A Python coding assistant integrated with Pippy to support children in learning programming in more efficient way. Pippy's AI-assistant uses a Llama3.1 model from Ollama with a [RAG architecture](https://arxiv.org/pdf/2005.11401) for retrieving relevent examples from the [documentations](https://github.com/sugarlabs/sugar-ai/tree/main/docs). Specially enhanced to handle Python, GTK, and Pygame queries, making it an essential tool for beginner programmers.
```

Comment:
Please make this into a complete sentence.

The Pippy Activity coding assistant is a gen-AI collaboration tool for children to learn and explore Python programming.